Geometry/GEMGeometryBuilder/plugins/GEMGeometryESModule.cc

```diff
@@ -20,13 +20,23 @@
 #include "FWCore/Framework/interface/ModuleFactory.h"
 #include "FWCore/Framework/interface/ESProducer.h"
 
+// Alignments
+#include "CondFormats/Alignment/interface/DetectorGlobalPosition.h"
+#include "CondFormats/Alignment/interface/AlignmentErrorsExtended.h"
+#include "CondFormats/AlignmentRecord/interface/GlobalPositionRcd.h"
+#include "CondFormats/AlignmentRecord/interface/GEMAlignmentRcd.h"
+#include "CondFormats/AlignmentRecord/interface/GEMAlignmentErrorExtendedRcd.h"
+#include "Geometry/CommonTopologies/interface/GeometryAligner.h"
+
 #include <memory>
 
 using namespace edm;
 
 GEMGeometryESModule::GEMGeometryESModule(const edm::ParameterSet & p)
+  : alignmentsLabel(p.getParameter<std::string>("alignmentsLabel"))
 {
   useDDD = p.getParameter<bool>("useDDD");
+  applyAlignment = p.getParameter<bool>("applyAlignment");
```

Comment:
all of the variables could be initialized in the constructor initialization list